Index: src/com/brevitaz/stage2/Statistics.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>package com.brevitaz.stage2;\n\n\n\npublic class Statistics {\n\tint total,firstHalfRow,secondHalfRow,firstHalfCount,secondHalfCount;\n\tpublic void getTotalIncome(int rowSize,int colmSize)\n\t{\n//\t\tSystem.out.println(rowSize+\" \"+colmSize);\n\t\ttotal=rowSize*colmSize;\n\t\tif(total>0 &&total<60)\n\t\t{\n\t\t\ttotal=rowSize*(colmSize*10);\n\t\t}\n\t\telse{\n\t\t\tfirstHalfRow=rowSize/2;\n\t\t\tsecondHalfRow=rowSize-firstHalfRow;\n\n\t\t\tfirstHalfCount = firstHalfRow *(colmSize*10) ;\n\t\t\tsecondHalfCount=secondHalfRow*(colmSize*8) ;\n\t\t\ttotal=firstHalfCount+secondHalfCount;\n\n\t\t}\n\t\tSystem.out.println(\"\\nTotal Income:\\n$\"+total);\n\n\t}\n}\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/com/brevitaz/stage2/Statistics.java b/src/com/brevitaz/stage2/Statistics.java
--- a/src/com/brevitaz/stage2/Statistics.java	
+++ b/src/com/brevitaz/stage2/Statistics.java	
@@ -23,5 +23,40 @@
 		}
 		System.out.println("\nTotal Income:\n$"+total);
 
+	}
+
+	public void getStatisticDetails(BookSeat _book)
+	{
+		rowCount=_book.hall.length;
+		colmCount= _book.hall[0].length;
+
+		total=rowCount*colmCount;
+		if(total>0 &&total<60)
+		{
+			total=rowCount*(colmCount*10);
+		}
+		else{
+			firstHalfRow=rowCount/2;
+			secondHalfRow=rowCount-firstHalfRow;
+
+			firstHalfCount = firstHalfRow *(colmCount*10) ;
+			secondHalfCount=secondHalfRow*(colmCount*8) ;
+			total=firstHalfCount+secondHalfCount;
+		}
+
+
+		for (row=0;row<rowCount;row++)
+		{
+			for (colm=0; colm< colmCount; colm++)
+			{
+				if(_book.hall[row][colm]=='B') {
+					numOfBooked++;
+				}
+			}
+		}
+		System.out.println("\nnumber of purchased tickets: $"+_book.getNumOfTickets());
+		System.out.println("\nCurrent Income:\n$"+_book.getCurrentIncome());
+		System.out.println("\nTotal Income:\n$"+total);
+
 	}
 }
